src/sources/soundsourceproxy.cpp

```diff
@@ -395,6 +395,7 @@ SoundSourceProxy::exportTrackMetadataBeforeSaving(
                 kLogger.warning()
                         << "Failed to update stream info from audio "
                            "source before exporting metadata";
+                return ExportTrackMetadataResult::Skipped;
```

Comment:
We need here adopt the change from: 
https://github.com/mixxxdj/mixxx/commit/a2b17efed1172810d6d2abcc8de646e0122c69cb
```suggestion
                return ExportTrackMetadataResult::Failed;
```

Reply:
Thanks, tests pass now (locally)